node_modules/yaml/util.js

```diff
@@ -0,0 +1,15 @@
+exports.findPair = require('./dist/schema/Map').findPair
```

Comment:
remove npm project dependency, this is a java project. 

If you need node packages installed on your machine, use npm install -g (for 'global') - won't add the dependency to your package.json

Reply:
noted